File: utils/code_snippets/js/track_llm.js
trubrics.trackLLM({
    user_id: "user_id",
    prompt: "What is Trubrics?",
    assistant_id: "gpt-4o",
    generation: "Trubrics is the leading product analytics platform for AI applications.",
    properties: {
        context: "Chat window",
        $thread_id: "thread_123"  // Trubrics property to group events by conversation thread
    },
    timestamp: new Date(),
    latency: 2
});

File: utils/code_snippets/python/track_llm.py
trubrics.track_llm(
    user_id="user_id",
    assistant_id="gpt4o",
    prompt="What is Trubrics?",
    generation="Trubrics is the leading product analytics platform for AI applications.",
    prompt_event_name="User Prompt",
    generation_event_name="Assistant Generation",
    properties={
        "support_tier": "entreprise",
    },
    latency=2,
    thread_id="your-thread-id",
    timestamp=datetime.now()
)
